gnocchi/rest/__init__.py

```diff
@@ -665,6 +665,9 @@ def post(self):
         except indexer.NoSuchResource as e:
             abort(404, e)
 
+        return pecan.request.indexer.get_resource(
```

Comment:
Not really. What I mean is that if the user wants the new updated resource he just have to do `GET /v1/resource/generic/<id>`.

And actually returning the entire resource on the `/v1/resource/generic/<id>/metric` seems wrong to me. This is not the resource endpoint, it's the metric of the resource endpoint. So if anything would be returned it's the new list of metric only – which would be a different SQL request, not sure there's such a call in the indexer API.


Also I did understand at first read you were trying to fix #12. I'd like to have @chungg opionion since he is the one that opened the bug, if this is really a requirement since it will cost an indexer request on each call and I'm not sure it's really worth it.